projects/js-packages/social-logos/tools/build-react

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
```

Comment:
> Finally, there's slightly better error-handling; the script returns a non-zero code if the build chain fails.

You may want to add `set -eo pipefail` to the top of the bash build files. That will make them automatically fail if some subcommand fails, versus now where they'll just continue on.

If you don't like that, every command that might fail would need a `|| exit 1`, or you'll want to chain everything together with `&&` like you did in https://github.com/Automattic/jetpack/blob/6a36c0fc86b4d8b36040187f2e0ca93e3d6c852f/projects/js-packages/social-logos/tools/build#L16-L23

Reply:
I've been burned by `pipefail` before (e.g. with `grep` or fast-and-loose exit codes from custom scripts), but in this case it should be fine: 16a333dba9